portfolio/src/main/webapp/index.html

```diff
@@ -1,52 +1,44 @@
 <!DOCTYPE html>
 <html lang="en">
 
-<head>
-	<meta charset="UTF-8">
-	<meta name="viewport" content="width=device-width, initial-scale=1">
-	<title>My Portfolio</title>
-	<link rel="stylesheet" href="style.css">
-	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.0/css/bulma.min.css">
-	<script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>
-	<script src="script.js"></script>
-
-</head>
-
-
-
-
-<body>
-	<section class="section">
-		<div class="container ">
-            <div class="notification is-info is-light">
-			    <h1 class="title"> Bhargavi Kumar Sundaresan <i class="fas fa-hand-spock"></i></h1>
-			    <p class="subtitle is-italic"> My portfolio!</p>
-                <br>
-            </div> 
-		</div>
-
-        <div><br></div>
-
-
-		<div class="container notification is-primary">
-			<article class="media">
-				<div class="media-left">
-					<figure class="image is-128x128"> <img src="/images/bks.jpg" alt="A picture of me!"> </figure>
-				</div>
-
-				<div class="media-content">
-					<div class="content">
-						<p>
-							<strong>A little ABOUT ME! </strong> <small> @bksundar@google.com </small>
-							<br>
-                            Hi there! I'm Bhargavi. I was born in LA, and moved to India when I was two years old. I grew up entirely in Chennai, Tamilnadu, and did my schooling in PSBB Nungambakkam. I moved to the US in 2018 to pursue my undergraduate degree in Computer Science. Currently, I am a rising junior at the University of California, Santa cruz.
-                        </p>
-					</div>
-				</div>
-
-			</article>
-		</div>
-	</section>
-</body>
-
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.0/css/bulma.min.css">
+    <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>
+    <script src="script.js"></script>
+  </head>
+
+  <body>
+    <section class="section">
+      <div class="container ">
+	    <div class="notification is-info is-light">
```

Comment:
It has been much better now. But the indentation is still messy starting from here.